components/console_simple_init/linker.lf

```diff
@@ -0,0 +1,13 @@
+[sections:plugins_desc]
+entries:
+    .plugins_desc
+
+[scheme:plugins_desc_default]
+entries:
+    plugins_desc -> flash_rodata
+
+[mapping:plugins_desc]
+archive: *
+entries:
+    * (plugins_desc_default);
```

Comment:
If you use the section name which an IDF example uses, i think you are inviting potential conflicts with users' code which is also copied from the same example :)

Could you change the section names and other identifiers to be specific to your component? (console_simple_init, console_simple_init_desc, console_simple_init_default, etc)

Reply:
Changed the section name to .console_cmd_desc.